mne/viz/tests/test_topo.py

```diff
@@ -296,6 +306,13 @@ def test_plot_topo_image_epochs():
     assert qm_cmap[0] is cmap
 
 
+def test_plot_topo_select():
+    """Test selecting sensors in an ERP topography plot."""
+    # Show topography
+    evoked = _get_epochs().average()
+    plot_evoked_topo(evoked, select=True)
```

Comment:
is this test unfinished? test title/docstring made me expect you were actually going to do lasso selection here.

Reply:
Added the proper test. Now I also test single channel picking. You can add/remove channels with a click by holding CTRL or CTRL+SHIFT